src/ids-toolbar/ids-toolbar-more-actions.js

```diff
@@ -0,0 +1,95 @@
+import { IdsElement, scss, customElement } from '../ids-base/ids-element';
+
+// @ts-ignore
+import styles from './ids-toolbar-more-actions.scss';
+
+// Subcomponents
+import IdsToolbarSection from './ids-toolbar-section';
+import IdsMenuButton from '../ids-menu-button/ids-menu-button';
+import IdsPopupMenu from '../ids-popup-menu/ids-popup-menu';
+
+/**
+ * IDS Toolbar Section Component
+ */
+@customElement('ids-toolbar-more-actions')
+@scss(styles)
+class IdsToolbarMoreActionsButton extends IdsElement {
```

Comment:
```suggestion
class IdsToolbarMoreActionsButton extends IdsElement {
```
A bit verbose? Maybe IdsToolbarActions? Not sure i guess its fine if no one really uses this?

Reply:
I've opted for `IdsToolbarMoreActions` since the component itself isn't just a "button", and the actual "actions" are in the other Toolbar sections (this contains "more" actions).